### PR TITLE
Bug Jira - get-issue with attachment wrong url

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/46579

## Description
Wrong url when trying to get attachment of an issue when calling command get-issue
image
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV2/JiraV2_test.py [hunk 1]
```diff
@@ -1282,3 +1282,23 @@ def test_get_custom_field_names(mocker, requests_mock):
...
+    """
+    from JiraV2 import get_issue
+    from test_data.raw_response import GET_ISSUE_WITH_ATTACHMENT_RESPONSE, MD_AND_CONTEXT_OUTPUT
+
+    mocker.patch.object(demisto, "params", return_value=integration_params)
+    mocker.patch('JiraV2.generate_md_context_get_issue', return_value=MD_AND_CONTEXT_OUTPUT)
+    requests_mock.get('https://localhost/rest/api/latest/issue/VIK-267', json=GET_ISSUE_WITH_ATTACHMENT_RESPONSE)
+    requests_mock.get('https://localhost/rest/api/2/attachment/content/16188', json={})
+
+    get_issue("VIK-267", get_attachments="true")  # the command will fail if not doing a request with the proper url
```
Comment (jochman) [inline Packs/Jira/Integrations/JiraV2/JiraV2_test.py:1304]:
```suggestion
    assert get_issue("VIK-267", get_attachments="true"), 'There was a request to the wrong url'
```

Much clearer when fails.

File: Packs/Jira/Integrations/JiraV2/JiraV2_test.py [hunk 2]
```diff
@@ -1282,3 +1282,23 @@ def test_get_custom_field_names(mocker, requests_mock):
     requests_mock.get('https://localhost/rest/api/latest/field', json=FIELDS_RESPONSE)
     res = get_custom_field_names()
     assert res == EXPECTED_RESP
+
+
+def test_get_issue_attachment_url(mocker, requests_mock):
```
Comment (jochman) [inline Packs/Jira/Integrations/JiraV2/JiraV2_test.py:1287]: Call the function in a describable name.

**Review by ShacharKidor — APPROVED** [review]

File: Packs/Jira/Integrations/JiraV2/test_data/raw_response.py [hunk 3]
```diff
@@ -241,23 +241,23 @@
...
+                   "avatarUrls": {
+                       "16x16": "",
+                       "24x24": "",
+                       "32x32": "",
+                       "48x48": ""
+                   },
+                   "displayName": "jon doe",
+                   "emailAddress": "admin@demistodev.com",
+                   "self": "https://demistodev.atlassian.net/rest/api/2/user?accountId=id",
+                   "timeZone": "Asia"
```
Comment (avidan-H) [inline Packs/Jira/Integrations/JiraV2/test_data/raw_response.py:258]: Be consistent with your quotation marks. They should either all be single quotes or double quotes - not a mixture of the two. The 'soon-to-be' official recommendation (it hasn't yet been added to the development conventions in the development docs yet) is to use single quotes.

File: Packs/Jira/Integrations/JiraV2/test_data/raw_response.py [hunk 4]
```diff
@@ -241,23 +241,23 @@
...
+               }
+               }
```
Comment (avidan-H) [inline Packs/Jira/Integrations/JiraV2/test_data/raw_response.py:260]: This bracket's placement is misformatted.

File: Packs/Jira/ReleaseNotes/1_4_8.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Fixed an issue where calling the command **get-issue** with *getAttachments="true"* returned an error.
```
Comment (avidan-H) [inline Packs/Jira/ReleaseNotes/1_4_8.md:4]:
```suggestion
- Fixed an issue where calling the **get-issue** command with *getAttachments="true"* caused an error.
```

**Review by avidan-H — CHANGES_REQUESTED** [review]

File: Packs/Jira/Integrations/JiraV2/test_data/raw_response.py [hunk 6]
```diff
@@ -182,82 +182,83 @@
...
+            'avatarUrls': {
+                '16x16': '',
+                '24x24': '',
+                '32x32': '',
+                '48x48': ''
+            },
+            'displayName': 'jon doe',
+            'emailAddress': 'admin@demistodev.com',
+            'self': 'https://demistodev.atlassian.net/rest/api/2/user?accountId=id',
+            'timeZone': 'Asia'
```
Comment (avidan-H) [inline Packs/Jira/Integrations/JiraV2/test_data/raw_response.py:259]: Looks like some of this info is real(ish)... make sure to use fake information.

**Review by avidan-H — CHANGES_REQUESTED** [review]

**Review by avidan-H — APPROVED** [review]

**Pull request merged**